### DISCUSSION TITLE
Water Leak Detector | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#823] [Q&A]
# Water Leak Detector | Seeed Studio Wiki

Water Leak Detector

https://wiki.seeedstudio.com/Sensor/SenseCAP/SenseCAP_Probe/Water-Leak-Detector/

**Comment by Miliks** [comment]
How should i connect this sensor to the Raspberry Pi 4B?
